plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/ui/AccountSelector.java

```diff
@@ -100,6 +100,10 @@ public int selectAccount(String email) {
     return index;
   }
 
+  public boolean isSignedIn() {
+    return loginService.hasAccounts();
```

Comment:
To be correct, this should be `getSelectedCredential() != null`. Currently, `AccountSelector` is not being synchronized with `loginService` by its design (#842). The parent of `AccountSelector` also relies on `getSelectedCredential()` eventually.